modules/javafx.graphics/src/main/java/com/sun/javafx/css/StyleManager.java

```diff
@@ -946,143 +936,14 @@ static byte[] calculateCheckSum(String fname) {
         return new byte[0];
     }
 
-    @SuppressWarnings("removal")
     public static Stylesheet loadStylesheet(final String fname) {
-        try {
-            return loadStylesheetUnPrivileged(fname);
-        } catch (java.security.AccessControlException ace) {
```

Comment:
As a note for other reviewers, @lukostyra and I discussed this offline, and determined that it was best to remove the `try` / `catch` along with the entire `catch` block. `AccessControlException` is no longer possible, so this was slated for removal as part of subsequent follow-up bug [JDK-8342998](https://bugs.openjdk.org/browse/JDK-8342998) already. By removing it as part of _this_ PR, it saves the effort of removing the `doPrivileged` calls and fixing up the unused permissions in dead code only to have them removed shortly afterwards.